data-reporting-service/person-service/src/main/java/gov/cdc/etldatapipeline/person/PersonServiceApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@ComponentScan(basePackages = {"gov.cdc.etldatapipeline"})
```

Comment:
do we require an explicit component scan  annotation, since it is in the same package

Reply:
the app didn't start without it! not able to recognize the classes from common util